stgit/commands/squash.py

```diff
@@ -1,3 +1,4 @@
+# TODO (not this file): add 'verbose=1' support to 'stg new', similar to 'git commit'
```

Comment:
This looks out of place.

Reply:
good catch! i've moved my TODO list to an external doc :p